README.md

```diff
@@ -54,26 +54,35 @@ To build for android, you need the Android SDK and platforms 21 and 26.  The eas
 - [SDK 26](https://developer.android.com/studio/releases/platforms#8.0)
 - [NDK 21](https://developer.android.com/ndk/downloads/)
 
-### 🌐 .NET
-
-A .NET Standard binding library is provided so that this package can be consumed from C# applications.  At this time, only Android and iOS binaries are included and they can be consumed from a Xamarin application.  Building for Xamarin currently requires a Mac
-
-- Other dependencies for Android and iOS
-- [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/)
-- [NuGet Command Line (CLI)](https://docs.microsoft.com/en-us/nuget/reference/nuget-exe-cli-reference#macoslinux)
-
 ### 🍏 iOS
 
 To build for iOS you need XCode installed
 
 - [XCode](https://developer.apple.com/xcode/resources/) and the [Command Line Tools for XCode](#)
 - [CMake 3.19](https://cmake.org/) may be necessary, along with changes to the Makefile.  [See ISSUE #138](https://github.com/microsoft/electionguard-cpp/issues/138)
 
+### Linux
+
+The automated install of dependencies is currently only supported on debian-based systems.  See the makefile for more information.
+
+### 🖥️ Windows (using MSVC)
+
+Building on windows is supported using the `MSVC` toolchain.
+
+- Install [Chocolatey](https://chocolatey.org/install)
+- Install [Powershell Core](https://github.com/powershell/powershell)
+- Install [VS 2019](https://visualstudio.microsoft.com/vs/)
+- Open the Visual Studio Installer and install
```

Comment:
i'm not entirely sure that this list is 100% accurate because of all of the stuff i ahve installed on my local machine.